tests/misc_env/install_standalone_osp.py

```diff
@@ -0,0 +1,275 @@
+import ipaddress
```

Comment:
Hi @hmaheswa , can we modularize the OSP installation, so we could enhance anytime with newer requirements.

Reply:
working on it @sunilkumarn417